src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-realm-form/rgw-multisite-realm-form.component.html

```diff
@@ -1,58 +1,78 @@
+  <form #frm="ngForm"
+        #formDir="ngForm"
+        [formGroup]="multisiteRealmForm"
+        novalidate>
+  <div cdsModalContent>
+    <div cdsCol
+    [columnNumbers]="{md: 4}">
+              <div class="form-item">
+                <cds-text-label for="realmName"
+                                cdRequiredField="Realm Name"
+                                [invalid]="!multisiteRealmForm.controls.realmName.valid && multisiteRealmForm.controls.realmName.dirty"
+                                [invalidText]="realmNameError"
+                                i18n>Username
+                  <input cdsText
+                         placeholder="Realm name..."
+                         i18n-placeholder
+                         id="realmName"
+                         name="realmName"
+                         formControlName="realmName"
+                         [invalid]="!multisiteRealmForm.controls.realmName.valid && multisiteRealmForm.controls.realmName.dirty"
+                         autofocus>
+                </cds-text-label>
+                <ng-template #realmNameError>
+                  <span *ngIf="multisiteRealmForm.showError('realmName', formDir, 'required')">
+                    <ng-container i18n>
+                     This field is required.
+                    </ng-container>
+                  </span>
+                  <span *ngIf="multisiteRealmForm.showError('realmName', formDir, 'uniqueName')">
+                    <ng-container i18n>
+                      The chosen realm name is already in use.
+                    </ng-container>
+                  </span>
+                </ng-template>
+              </div>
+              <div class="bx--form-item inline-helper">
+        <cds-checkbox 
+                      id="default_realm"
                    name="default_realm"
                    formControlName="default_realm"
-                   [attr.disabled]="action === 'edit' ? true: null"
-                   type="checkbox">
-            <label class="form-check-label"
-                   for="default_realm"
-                   i18n>Default</label>
-            <cd-helper *ngIf="action === 'edit' && info.data.is_default">
-              <span i18n>You cannot unset the default flag.</span>
-            </cd-helper>
-            <cd-helper *ngIf="action === 'edit' && !info.data.is_default">
-              <span i18n>Please consult the&nbsp;<a href="{{ docUrl }}">documentation</a>&nbsp;to follow the failover mechanism</span>
-            </cd-helper>
-            <cd-helper *ngIf="defaultRealmDisabled && action === 'create'">
-              <span i18n>Default realm already exists.</span>
-            </cd-helper>
-          </div>
-        </div>
+                   [attr.disabled]="initialObjData.action === 'edit' ? true: null"
+                      i18n>Default
+        </cds-checkbox>
+        <cd-helper *ngIf="initialObjData.action === 'edit' && initialObjData.info.data.is_default" class="helper-text">
```

Comment:
We need to replace this too with carbonized info tip, thanks

Reply:
cd-helper internally using cds-tooltp.